.pnpmfile.cjs

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  hooks: {
+    afterAllResolved: async lockfile => {
+      // Object.keys(lockfile.packages)
```

Comment:
Uhm, leftover?

Reply:
Nope, I want to keep this for the follow up, because this code will most likely be needed for it.